### PR TITLE
Ventricular 2D Segmentation Network

### PR DESCRIPTION
**Pull request by ericspod**
Signed-off-by: Eric Kerfoot <eric.kerfoot@kcl.ac.uk>

Fixes # .

### Description
This is a UNet for segmenting the heart in 2D short axis MR images. This was trained in essentially this configuration with old pre-MONAI code.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json`.
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.

### CODE REVIEW

File: models/ventricular_short_axis_3label/configs/train.json [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    "trainer": {
+        "_target_": "SupervisedTrainer",
+        "max_epochs": "@num_epochs",
+        "device": "@device",
+        "train_data_loader": "@train_dataloader",
+        "network": "@network",
+        "loss_function": "@lossfn",
+        "optimizer": "@optimizer",
+        "postprocessing": "@post_transform",
+        "key_train_metric": "$None",
```
Comment (Nic-Ma) [inline models/ventricular_short_axis_3label/configs/train.json]: I think JSON accepts `null` as `None` in the python.

Thanks.

**Comment by Nic-Ma** [comment]
Hi @ericspod ,

To keep the repo simpler, I think maybe it's better to put the model.pt/ts and `SC-N-2-3-0.npy` in large files?

Thanks.
